games/arma3/entrypoint.sh

```diff
@@ -392,10 +392,16 @@ if [[ ${HC_NUM} > 0 ]]; then
     done
 fi
 
+# Define the log file path with a timestamp
+LOG_FILE="/home/container/logs/arma3server_$(date '+%m_%d_%Y_%H%M%S').log"
```

Comment:
Would it be better to have it be `.rpt` instead of `.log`, just to help differentiate it between the `.log` files in `serverprofile/`?

Reply:
Yes, I'll change that.